calico/_includes/release-notes/v3.23.0-release-notes.md

```diff
@@ -0,0 +1,145 @@
+09 May 2022
+
+#### IPv6 VXLAN support
+
+Calico now supports VXLAN encapsulation for IPv6 networks.
+
+Pull Requests:
+
+ - Add IPv6 VXLAN support [calico #5988](https://github.com/projectcalico/calico/pull/5988) (@coutinhop)
+ - Add IPv6 VXLAN route calculation support to Felix [calico #5919](https://github.com/projectcalico/calico/pull/5919) (@coutinhop)
+ - Allocate IPv6 VXLAN tunnel addresses [calico #5893](https://github.com/projectcalico/calico/pull/5893) (@coutinhop)
+
+#### VPP dataplane beta
+
+The Calico VPP dataplane integration is now in beta!
+
+#### Calico networking support in AKS
+
+You can now use the Calico CNI in your AKS clusters to take advantage of all of the Calico networking features.
+
+Pull Requests:
+
+ - Updated Calico doc for AKS Bring Your Own CNI. [calico #5857](https://github.com/projectcalico/calico/pull/5857) (@song-jiang)
+
+#### BGP enhancements
+
+We have added additional configuration options to allow for more control in specific scenarios, such as node-to-node mesh and separate systems running Calico using the same AS.
```

Comment:
> such as node-to-node mesh and separate systems running Calico using the same AS.

This makes it sounds like we introduced a mesh mode, as well as the ability to use different AS numbers.

We should spell out the specific changes and use-cases for them in more detail. There's no problem being a bit wordy here - we want to explain the value of what we did.